state-chain/runtime/src/migrations/api_calls_gas_migration.rs

```diff
@@ -0,0 +1,195 @@
+use crate::*;
+use cf_chains::{
+	eth::api::{register_redemption, update_flip_supply},
+	evm::api::{
+		all_batch, common::EncodableTransferAssetParams, execute_x_swap_and_call,
+		set_agg_key_with_agg_key, set_comm_key_with_agg_key, set_gov_key_with_agg_key,
+		transfer_fallback, EvmTransactionBuilder,
+	},
+};
+use frame_support::traits::UncheckedOnRuntimeUpgrade;
+use sp_std::marker::PhantomData;
+
+#[cfg(feature = "try-runtime")]
+use sp_runtime::DispatchError;
+#[cfg(feature = "try-runtime")]
+use sp_std::{vec, vec::Vec};
+
+pub mod old {
+
+	use super::*;
+	use cf_chains::{
+		evm::{
+			api::{EvmReplayProtection, SigData},
+			AggKey,
+		},
+		Chain,
+	};
+	use codec::{Decode, Encode, MaxEncodedLen};
+	use frame_support::{CloneNoBound, DebugNoBound, EqNoBound, Never, PartialEqNoBound};
+	use scale_info::TypeInfo;
+	use sp_core::RuntimeDebug;
+
+	#[derive(Encode, Decode, TypeInfo, Clone, RuntimeDebug, PartialEq, Eq)]
+	pub struct ExecutexSwapAndCall {
+		pub transfer_param: EncodableTransferAssetParams,
+		pub source_chain: u32,
+		pub source_address: Vec<u8>,
+		pub gas_budget: <Ethereum as Chain>::ChainAmount,
+		pub message: Vec<u8>,
+	}
+
+	#[derive(Encode, Decode, TypeInfo, MaxEncodedLen, Clone, RuntimeDebug, PartialEq, Eq)]
+	pub struct EvmTransactionBuilder<C> {
```

Comment:
I think we dont need to define this here and can use the existing one since it did not change